org.eclipse.scanning.test/src/org/eclipse/scanning/test/event/DeviceRequestTest.java

```diff
@@ -141,21 +159,21 @@ public void simpleSerialize() throws Exception {
 
 	// @Test
 	public void testGetDevices() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)